src/scripts/main.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('span');
```

Comment:
```suggestion
  const notification = document.createElement('div');
```
span це строковий елемент, і він семантично не має в себе включати блоки h2 чи p (які є блочними елементами), використовуй div в таких випадках